src/main/webapp/js/instructorFeedbackEdit.es6

```diff
@@ -955,9 +955,13 @@ function setTooltipTriggerOnFeedbackPathMenuOptions() {
     });
 }
 
+function enableOnclicks() {
+    $(document).on('click', '.enable-edit-fs', () => enableEditFS());
+}
+
 $(document).ready(() => {
     prepareInstructorPages();
-
+    enableOnclicks();
```

Comment:
Over here it actually violates SLAP if I throw it into `$(document).ready`. Maybe `readyFeedbackEditPage` would work...